src/lib/Backpack/devBackpack.cpp

```diff
@@ -40,10 +40,11 @@ void startPassthrough()
     uint32_t baud = PASSTHROUGH_BAUD == -1 ? BACKPACK_LOGGING_BAUD : PASSTHROUGH_BAUD;
     // get ready for passthrough
     if (GPIO_PIN_RCSIGNAL_RX == GPIO_PIN_RCSIGNAL_TX)
-    {
-        // if we have a single S.PORT pin for RX then we assume the standard UART pins for passthrough
-        CRSF::Port.begin(baud, SERIAL_8N1, 3, 1);
-    }
+        #if defined(PLATFORM_ESP32_S3)
```

Comment:
Lets keep the open/close curly braces, as thats the style we are going for.
Also, can we have the comments in English please.

Reply:
Sorry, I forgot to change it